src/common/Widget.js

```diff
@@ -171,13 +171,15 @@
                     }
                     break;
                 case "html-color":
-                    var litmus = 'red';
-                    var d = document.createElement('div');
-                    d.style.color=litmus;
-                    d.style.color=_;
-                    //Element's style.color will be reverted to litmus or set to '' if an invalid color is given
-                    if( _ !== litmus && (d.style.color === litmus || d.style.color === '')){
-                        console.log("Invalid value for '" + id + "':  " + _);
+                    if (window.__hpcc_debug && _ && _ !== "red") {
```

Comment:
IMO The test is too expensive to be used at runtime every time a color is set (specifically the creation of the DOM node).  This way we can enable the test in debug mode for the unit testing and / or other testing.  

Also at runtime it doesn't change the flow of the program, it just spits out a console message.